### DISCUSSION TITLE
A JavaScript error occurred in the main process ERROR

### DISCUSSION THREAD

**Discussion by julianosro** [spacemeshos/go-spacemesh#5955] [Q&A]
this is what I receive when I try to open the spacemesh app on windows can anyone help please

![WhatsApp Image 2024-05-19 at 10 45 38_18986205](https://github.com/spacemeshos/go-spacemesh/assets/132822675/7a61cd5c-4999-4469-8f4f-26b85354aed3)
![WhatsApp Image 2024-05-19 at 10 45 38_fa578ff2](https://github.com/spacemeshos/go-spacemesh/assets/132822675/94175818-bbe6-4141-bef8-e201c9639f37)

**Comment by brusherru** [comment]
Hello!
Could you please share app-logs?

Due to screenshots, it seems like you have a corrupted node-config.json or some other JSON file. So please try to remove node-config.json and run Smapp again.

**Reply by julianosro** [reply]
hi i can't even get the log files even uninstalled the app and reinstalled it and exactly the same issue
